tests/basic.test.res

```diff
@@ -1,10 +1,10 @@
 open Js
 open Vitest
 
-test("Math.sqrt()", t => {
+test("Math.sqrt()", () => {
   open Expect
 
-  t->assertions(3)
```

Comment:
This `t` object will be used in future release as [test context](https://vitest.dev/guide/test-context.html).

Reply:
Do you want me to keep it around then for now? From reading the test context page `t->assertions(3)` still wouldn't work (since that's not called on the test context). But happy to make sure people have to still start out with `_ => {}` in their tests.

So I think introducing test context was already a breaking change and now still is a breaking change.

I could give it an opaque type but I think the pattern that Vitest uses here is difficult to type in ReScript? So not sure I can already fully tackle the problem of making it test context.

Reply:
> From reading the test context page t->assertions(3) still wouldn't work (since that's not called on the test context).

Currently all assertions/expects work correctly only on implicit global context because they are not bound to the test context correctly.

So it seems like we have to choose between binding the test context correctly here or vs supporting only global context at zero cost.